web-ui/view/github.ml

```diff
@@ -228,13 +228,26 @@ let cancel_success_message success =
   | [] -> div [ span [ txt @@ Fmt.str "No jobs were cancelled." ] ]
   | success -> ul (List.map format_job_info success)
 
-let cancel_success_message_v1 success =
-  let format_job_info ji =
-    (`Success, Fmt.str "Cancelling job: %s" ji.Client.variant)
+let success_message_v1 action jis =
+  let empty_message, prefix =
+    match action with
+    | `Cancel -> ("No jobs were cancelled.", "Cancelling")
+    | `Rebuild -> ("No jobs were rebuilt.", "Rebuilding")
   in
-  match success with
-  | [] -> [ (`Success, Fmt.str "No jobs were cancelled.") ]
-  | success -> List.map format_job_info success
+  match jis with
+  | [] -> [ (`Success, empty_message) ]
+  | jis ->
+      let trimmed = List.filteri (fun i _ -> i < 5) jis in
+      let message =
+        Astring.String.concat ~sep:"\x0C, "
```

Comment:
Any reason for using \x0C in the separator? Seems like it's just a control character for printers (form feed).

Reply:
Oh hmm cruft left over from trying to format the text. Thanks for catching.